src/App.jsx

```diff
@@ -14,70 +16,53 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selected, selectedGood] = useState('Jam');
```

Comment:
useState have have a array with two items:
- first item is a current value of state, so it's selectedGood
- second item if callback for change current value in a state, so it should be the same name a value, but with prefix **set**
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```